hazelcast-sql/src/test/java/com/hazelcast/jet/sql/SqlImposeOrderFunctionTest.java

```diff
@@ -295,4 +297,20 @@ private static String createTable(Object[]... values) {
         );
         return name;
     }
+
+    public static LocalTime time(long epochMillis) {
+        return timestampTz(epochMillis).toLocalTime();
+    }
+
+    public static LocalDate date(long epochMillis) {
+        return timestampTz(epochMillis).toLocalDate();
+    }
+
+    public static LocalDateTime timestamp(long epochMillis) {
+        return timestampTz(epochMillis).toLocalDateTime();
+    }
+
+    public static OffsetDateTime timestampTz(long epochMillis) {
+        return OffsetDateTime.ofInstant(Instant.ofEpochMilli(epochMillis), ZoneOffset.UTC);
+    }
```

Comment:
Those don't need to be public. However, I envision them being used in other tests as well but in that case they should be moved.